### PR TITLE
Load data from datastore and add that data to a local list.  

### PR DESCRIPTION
**Pull request by josefj1519**
Assignee: LiRq95

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,27 +31,28 @@
...
 public class DataServlet extends HttpServlet {
-  private ConcurrentSkipListSet<String> shows = new ConcurrentSkipListSet<String>();
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Show").addSort("timestamp", SortDirection.DESCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+    List<String> shows = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      shows.add((String)entity.getProperty("show"));
```
[the remark this answers is not in the document]
Comment (josefj1519) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Fixed and for loop will be later needed.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -38,7 +38,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
     List<String> shows = new ArrayList<>();
     for (Entity entity : results.asIterable()) {
-      shows.add((String)entity.getProperty("show"));
+      shows.add(entity.getProperty("show").toString());
```
Comment (LiRq95) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:42]: No need to even add the toString, that's already implicitly called when it realizes that it's trying to be set as a String.
So just `shows.add(entity.getProperty("show"));`

Comment (josefj1519) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:42]: Get an error doing that.  'java.lang.Object cannot be converted to java.lang.String'

Comment (LiRq95) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:42]: That's interesting... Definitely not the expected behavior. Alright we can leave the toString in there.
Normally, Objects and its inheritors have their toString behave as shown in this [article](https://www.baeldung.com/java-tostring). Not too sure what the exception is here.

**Pull request merged**